### PR TITLE
fix: remove link to unsafe resource & fix text mistakes in Ukrainian list

### PR DESCRIPTION
**Pull request by MaximFedarau**
## What does this PR do?
Remove resource(s) & Add info

## For resources:
Language: Ukrainian.
- Add resource for React.
- Check all document.

### Why is this valuable (or not)?
- I remove link to an unsafe resource + make text more readable.
- Add React section + React resources.

### How do we know it's really free?
It's a link to translated React docs.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

### CODE REVIEW

**Comment by MaximFedarau** [comment]
@eshellman

**Comment by MaximFedarau** [comment]
@eshellman

**Comment by MaximFedarau** [comment]
@eshellman

File: books/free-programming-books-uk.md [hunk 1]
```diff
@@ -18,28 +19,32 @@
...
 ### Haskell
 
-* [Вивчити собі Хаскель на велике щастя!](http://haskell.trygub.com) - Міран Ліповача
+* [Вивчить собі Хаскела на велике щастя!](http://haskell.trygub.com) - Міран Ліповача
 
 
 ### Java
 
-* [Програмування на Java для дітей, батьків, дідусів та бабусь](http://myflex.org/books/java4kids/java4kids.htm) - Яків Файн
+* [Програмування мовою Java для дітей, батьків, дідусів та бабусь](http://myflex.org/books/java4kids/java4kids.htm) - Яків Файн
```
Comment (eshellman) [inline books/free-programming-books-uk.md:27]: this doesn't seem to be a ukrainian language resource

File: books/free-programming-books-uk.md [hunk 2]
```diff
@@ -18,28 +19,32 @@
...
 
 
 ### JavaScript
 
 * [Розуміння ECMAScript 6](http://understandinges6.denysdovhan.com) - Денис Довгань (LambdaBooks)
 
 
+#### React
+
+* [Документація React](https://uk.reactjs.org/docs/getting-started.html)
```
Comment (eshellman) [inline books/free-programming-books-uk.md:37]: Don't invent titles! Use the title given by the resource. title is "Початок роботи" right?

**Review by eshellman — CHANGES_REQUESTED** [review]
other changes seem correct

File: books/free-programming-books-uk.md (hunk 2 above)
Comment (MaximFedarau) [reply books/free-programming-books-uk.md:37]: Yeah. Changed title.

File: books/free-programming-books-uk.md (hunk 1 above)
Comment (MaximFedarau) [reply books/free-programming-books-uk.md:27]: Ok. Removed it.

**Comment by MaximFedarau** [comment]
I think, that I fixed everything, @eshellman.

**Comment by MaximFedarau** [comment]
I think, that I fixed everything, @eshellman.

**Pull request merged**